src/handlers/swap_handlers.ts

```diff
@@ -166,6 +177,13 @@ const parseGetSwapQuoteRequestParams = (req: express.Request): GetSwapQuoteReque
             ? undefined
             : parseStringArrForERC20BridgeSources(req.query.excludedSources.split(','));
     const affiliateAddress = req.query.affiliateAddress;
+    const rfqt =
+        req.query.intentOnFilling === undefined
+            ? undefined
+            : { intentOnFilling: req.query.intentOnFilling === 'true' ? true : false };
```

Comment:
Nit but can't this just be `{ intentOnFilling: req.query.intentOnFilling === 'true' }`

Reply:
Addressed in https://github.com/0xProject/0x-api/pull/162/commits/735fe3e934b3f27397f09d8f0644a8c8271a6210